10/Dockerfile.rhel8

```diff
@@ -0,0 +1,72 @@
+FROM rhel8/s2i-base
+
+# This image provides a Node.JS environment you can use to run your Node.JS
+# applications.
+
+EXPOSE 8080
+
+# Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
+# available on the CLI without using npm's --global installation mode
+# This image will be initialized with "npm run $NPM_RUN"
+# See https://docs.npmjs.com/misc/scripts, and your repo's package.json
+# file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+
+ENV NODEJS_VERSION=10 \
+    NPM_RUN=start \
+    NAME=nodejs \
+    NPM_CONFIG_PREFIX=$HOME/.npm-global \
+    PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH
+
+ENV SUMMARY="Platform for building and running Node.js $NODEJS_VERSION applications" \
+    DESCRIPTION="Node.js $NODEJS_VERSION available as container is a base platform for \
+building and running various Node.js $NODEJS_VERSION applications and frameworks. \
+Node.js is a platform built on Chrome's JavaScript runtime for easily building \
+fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model \
+that makes it lightweight and efficient, perfect for data-intensive real-time applications \
+that run across distributed devices."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Node.js $NODEJS_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,$NAME,${NAME}${NODEJS_VERSION}" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="${APP_ROOT}/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
+      com.redhat.component="${NAME}-${NODEJS_VERSION}-container" \
+      name="rhel8/$NAME-$NODEJS_VERSION" \
+      version="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-nodejs-container" \
+      usage="s2i build <SOURCE-REPOSITORY> rhel8/$NAME-$NODEJS_VERSION:latest <APP-NAME>"
+
+RUN yum -y module enable nodejs:$NODEJS_VERSION && \
+    INSTALL_PKGS="nodejs npm nodejs-nodemon nss_wrapper" && \
+    ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon && \
+    yum remove -y $INSTALL_PKGS && \
```

Comment:
I guess the same should be also in 8/Dockerfile.rhel8. I know that now nodejs:8 is used in s2i-base, but in the future it could be changed and in that case Dockerfile for nodejs would need to be changed too.

Reply:
Makes sense, but maybe `dnf distrosync` would be better here so that we only change the nodejs modular rpms.

Reply:
Unfortunately `dnf distrosync` does not really work for Fedora based images due to many changes from the base image so I have left the `yum remove -y $INSTALL_PKGS` command in all Dockerfile version to have as few differences as possible.